### ISSUE TITLE
[IMPROVE] Shit SPAM.

### ISSUE THREAD

**Issue by SinSiXX** [OpenBB-finance/OpenBBTerminal#1625]
**Stop your shit SPAM.**

`Hi,  We, as OpenBB, would like to personally thank you for signing up to our newsletter.  We are on a mission to make investment research effective, powerful and accessible to everyone.  The team would love to hear what you think of our OpenBB Terminal and if there is anything we can do to improve it.  Feel free to e-mail us at hello@openbb.co or reach us through our Discord to get involved with the community.  Investment just evolved 🦋`

Ya ok, shove that butterfly up your fat ass.
Don't at me, just stop your bullshit.

**Comment by DidierRLopes** [comment]
Hi @SinSiXX,

This was a mistake from our side. The idea was to reach out to everyone and let them know:
   1. That we changed name of organization and repository. 
   2. That we are funded.

Here is the e-mail we had prepared. And as you can see the idea was not to leave anyone as subscriber but give you the chance to subscribe.

Sorry for this.

```
Hey there,

You must be wondering who is OpenBB and who am I. My name is Didier Lopes and I’m the developer that started Gamestonk Terminal. A project that you have starred on my GitHub page and this is how I’m able to reach out.

I hope you don't mind me reaching out directly by email and this will only happen once, but I wanted to give you an update about our name change and share with you our exciting journey, which you can find here: https://openbb.co/blog/gme-didnt-take-me-to-the-moon-but-gamestonk-terminal-did

TL;DR. We are on a mission to make investment research effective, powerful and accessible for everyone.
If you are interested in learning more about us, you can subscribe https://openbb.co/newsletter. Otherwise, thank you for your support in helping us achieve this goal and for following the terminal.

Thank you for your understanding, have a great week!
Didier Lopes
```

**Comment by SinSiXX** [comment]
Sorry, I got mad. Thx.

**Comment by jgreever** [comment]
Seems a little illegal to scrape our email addresses then sign us up for a mailing list that we never authorized. It's time to let Mailchip know about this harassment now. Contact Mailchip and send the email with headers to them now. They are lying about it being a mistake on their side, they knew what they were doing, do not let them get away with this crap.

`

Welcome to OpenBB Newsletter | Welcome to OpenBB Newsletter
-- | --
Welcome to OpenBB Newsletter

Welcome to OpenBB Newsletter
Hi,

We, as OpenBB, would like to personally thank you for signing up to our newsletter.

We are on a mission to make investment research effective, powerful and accessible to everyone.

The team would love to hear what you think of our OpenBB Terminal and if there is anything we can do to improve it.

Feel free to e-mail us at [hello@openbb.co](mailto:hello@openbb.co) or reach us through [our Discord](https://finance.us14.list-manage.com/track/click?u=e62ea2c860d2a0c3f3265c725&id=a573e97cc7&e=395ac15eb2) to get involved with the community.

Investment just evolved [🦋](https://finance.us14.list-manage.com/track/click?u=e62ea2c860d2a0c3f3265c725&id=145eada420&e=395ac15eb2)`

**Comment by DidierRLopes** [comment]
@jgreever

The team wanted to let users know that a project they starred has completely been renamed. This is because, personally, I would like to know if a project I had starred completely rebranded.

We only had the e-mail shown above explaining that we are the previous `GamestonkTerminal` on the queue. The e-mail you received was automatically sent by Mailchimp, it was an accident. See below,

<img width="635" alt="Screenshot 2022-04-03 at 22 16 12" src="https://user-images.githubusercontent.com/25267873/161450665-8f15c938-73f1-498e-91e0-9a772d5a64a1.png">

We have already deleted the entire audience from Mailchimp and will not contact anyone else that doesn't need to be contacted.

![image](https://user-images.githubusercontent.com/25267873/161450691-7d2c3b18-0479-45ad-bd94-84ff30161e8f.png)

I know this is frustrating, and am really sorry about this.

**Comment by fugkco** [comment]
> The team wanted to let users know that a project they starred has completely been renamed

This is insane. If I star a project on GitHub, it will stay starred when the org/repo is renamed. Why do you need to reach out to individuals about it? It's clear you tried scraped user's emails without people's consent so you have an audience to reach.

**Comment by DidierRLopes** [comment]
> > The team wanted to let users know that a project they starred has completely been renamed
> 
> This is insane. If I star a project on GitHub, it will stay starred when the org/repo is renamed. Why do you need to reach out to individuals about it? It's clear you tried scraped user's emails without people's consent so you have an audience to reach.

It has happened to me that I followed someone on instagram and they sold their account and then I was following something completely different.

This was my thinking, but I realize now that not everyone thinks the same. So I apologise from our end.

We've already deleted the entire audience. Let me know if there's anything I can do for you @fugkco
